featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -72,11 +72,9 @@ def calculate_feature_matrix(features, entityset=None, cutoff_time=None, instanc
         instance_ids (list): List of instances to calculate features on. Only
             used if cutoff_time is a single datetime.
 
-        dataframes (dict[str -> tuple(DataFrame, str, str,
-                                        dict[str -> str/Woodwork.LogicalType],
-                                        dict[str->str/set],
-                                        boolean)]): dictionary of DataFrames.
-            Entries take the format dataframe name -> (dataframe, index column, time_index, logical_types, semantic_tags, make_index)}.
+        dataframes (dict[str -> tuple(DataFrame, str, str, dict[str -> str/Woodwork.LogicalType], dict[str->str/set], boolean)]): 
+            Dictionary of DataFrames. Entries take the format 
```

Comment:
Putting all of this on one line was the best way I found of fixing the formatting warning that was showing up here and in the other functions where we have this param. The other option, from what I could tell, was to put all the newline elements of this tuple at the same level as the rest of the definition, which I didn't love, but maybe it's better than the super long line.